docs/src/pages/components/DateField/_DateField.docs.scss

```diff
@@ -0,0 +1,54 @@
+/*
+Date field
+
+@uswds https://designsystem.digital.gov/components/form-controls/#date-input
+
+Three text fields are the easiest way for users to enter most dates.
+
+Markup: datefield.example.html
+
+Style guide: components.date-field
+*/
+
+/*
```

Comment:
I'm my testing it looks like we only need to keep the top portion of this page and the rest will be inherited from the core CMS Design System. I think you can remove line 13 to the end and push the PR back up

Reply:
Line 13 and beyond has been removed and ready for review